creativecloud/features/firefly/firefly-interactive.css

```diff
@@ -0,0 +1,149 @@
+.hide {
+  display: none
+}
+
+.prompt-text::placeholder {
+  color: #2c2c2c;
+}
+
+.prompt-text.light::placeholder {
+  color: #dbdbdb;
+}
+
+.firefly-prompt {
+  position: absolute;
+  top: 16px;
+  margin-inline-start: 16px;
+  margin-inline-end: 16px;
+}
+
+.genfill-promptbar {
+  position: absolute;
+  top: 24px;
+  margin-inline-start: 16px;
+  height: 30px;
+  display: flex;
+}
+
+.interactive-marquee .media {
+  top: 72px;
+}
+
+.interactive-marquee .media picture img,
+.interactive-marquee .media video {
+  border-radius: 16px;
+}
+
+@media screen and (min-width: 1200px) {
+  .firefly-prompt {
+    top: 40px;
+    margin-inline-start: 40px;
+    margin-inline-end: 40px;
+  }
+
+  .genfill-promptbar {
+    top: 56px;
+    margin-inline-start: 40px;
+    height: 50px;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 1199px) {
+  .firefly-prompt {
+    top: 32px;
+    margin-inline-start: 32px;
+    margin-inline-end: 32px;
+  }
+
+  .genfill-promptbar {
+    top: 40px;
+    margin-inline-start: 32px;
+    height: 40px;
+  }
+
+  .firefly-selectortray {
+    top: 0;
+    margin-inline-start: 42px;
+    margin-inline-end: 42px;
+    width: 80%;
+  }
+
+  .interactive-marquee .interactive-container {
+    height: 813px;
+  }
+}
+
+@media (max-width: 600px) {
+  .firefly-selectortray {
+    min-width: 110%;
+    margin-inline-start: -22px;
+    margin-inline-end: 16px;
+    justify-content: center;
+    margin-top: -7px;
+    margin-bottom: 24px;
+  }
+
+  .interactive-marquee .interactive-container {
```

Comment:
We also need to add max-width of .interactive-marquee .container to 100% to avoid conflict with default AEM max-width.